### DISCUSSION TITLE
Changes to RDF system to address low server population problem

### DISCUSSION THREAD

**Discussion by ProhorovIlja** [chromiecraft/chromiecraft#6881] [Suggestions]
At the moment RDF for low level players is pretty much dead.
Because you can sit in a queue waiting for a group to fill in for a very, very long time.
Veritas server added an auto-balance, or something, which allows players to do dungeons in solo with increased stats, and they will be able to complete them alone.
What if in the future CC could add a feature, which allows players, who've been sitting in a queue for 5–10 minutes to go solo instead, if a dungeon group hasn't been filled.
An alternative could've been to add bots, but bots will put an extra load on a server.
So auto-balance for a solo run after an x amount of time passed, and no group has been found might be a perfect solution.

**Comment by elthehablo** [comment]
This would require drastic changes, as well as use of a specific module but also a custom implementation of that module to work. I don't think we can do anything with this, sorry.

Bots are 100% out of the question.
